public/app/plugins/datasource/prometheus/components/PromExemplarField.tsx

```diff
@@ -25,6 +25,13 @@ export function PromExemplarField({ datasource, onChange, isEnabled, refId }: Pr
     };
   }, [datasource, refId]);
 
+  // If exemplars are disabled, set them to false
+  useEffect(() => {
+    if (error) {
+      onChange(false);
```

Comment:
We can probably put it into the previous effect where we are setting the error on line 21. Does not seem like there is a need for separate effect.